core/mono-samples/iOS/README.md

```diff
@@ -0,0 +1,25 @@
+---
+languages:
+- c#
+products:
+- dotnet
+- iOS
+- mono runtime
```

Comment:
Right now this needs to be _dotnet-core_ so that it appears in the filter list for the samples browser. There isn't a product filter for _ios_ and _mono runtime_ unfortunately. We'll need to do something about this in the future though: https://review.docs.microsoft.com/en-us/samples/browse/?branch=master&expanded=dotnet&products=dotnet-core

```suggestion
products:
- dotnet-core
```

Reply:
Will there be dotnet5 or dotnet6 added to the product filters in the future?

Reply:
No idea. I don't have anything to do with the samples. I'll bring this up to our team and see what they know.